iceberg/pom.xml

```diff
@@ -74,6 +74,16 @@
         <groupId>org.apache.iceberg</groupId>
         <artifactId>iceberg-core</artifactId>
         <version>${iceberg.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>org.apache.httpcomponents.client5</groupId>
+            <artifactId>httpclient5</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>org.apache.httpcomponents.core5</groupId>
+            <artifactId>httpcore5</artifactId>
+          </exclusion>
+        </exclusions>
```

Comment:
What happens if we don't exclude it? We eventually need this dependency when we adapt Hive QL to Iceberg REST API. It sounds fine to exclude it at this point.